code/wdb/datastripe.q

```diff
@@ -59,6 +59,32 @@ initdatastripe:{
 
 \d .ds
 
+symlink:{
+    /- function to create HDB sym file and symlink to this sym file at start up
+    /- create HDB sym file
+    symdir:`$raze string .wdb.hdbdir,"/sym";
+    if[not `sym in key .wdb.hdbdir;symdir set `symbol$()];
+    .lg.o[`hdbsym;"creating HDB symfile"];
```

Comment:
I think the log should probably be before the action - at the moment by the time it logs, the file is already created